src/components/sidebar/index.tsx

```diff
@@ -1,4 +1,5 @@
 import { useMemo, useState } from 'react'
+import slugify from 'slugify'
```

Comment:
FYI this removes `/`, doesn't replace them with `-`, which is why I didn't end up using it. I think we need to discuss this but okay to do that outside of this PR, just want to discuss soon after.